docs/source/quick-start.ipynb

```diff
@@ -604,7 +604,7 @@
      "name": "stderr",
```

Comment:
<p>Might be worth putting a markdown cell here explaining why you would use one over the other. As it stands, someone would ask why not just use the fast version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xskillscore/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>